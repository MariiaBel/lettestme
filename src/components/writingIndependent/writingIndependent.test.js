import React from "react";
import WritingIndependent from "./writingIndependent.jsx";
import renderer from "react-test-renderer";

it(`Should second part of writing section render correctly`, () => {
    const tree = renderer
        .create(<WritingIndependent
            timeToFinish={20}
            onWritingQuestionClick={() => { }}
        />)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
