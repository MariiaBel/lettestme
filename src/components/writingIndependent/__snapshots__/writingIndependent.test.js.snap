// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should second part of writing section render correctly 1`] = `
<article>
  <header>
    <h1>
      TOEFL. Independent Writing
    </h1>
  </header>
  <span>
    TIMER: 
    20
  </span>
  <form>
    <textarea>
       
    </textarea>
    <button
      onClick={[Function]}
    >
      Next
    </button>
  </form>
</article>
`;
